<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-DGJD4V4" name="Default test" tests="23" failures="5" timestamp="31 May 2018 08:14:13 GMT" time="0.727" errors="0">
  <testcase name="postGetUserAuthDetailsTestCorrectParameter" time="0.067" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsDecimal" time="0.009" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsEmpty" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsError" time="0.03" classname="com.userManagement.GetUserAuthDetailsTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[-1]&amp;gt; but was:&amp;lt;[0]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[-1]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.userManagement.GetUserAuthDetailsTest.postGetUserAuthDetailsTestUidIsError(GetUserAuthDetailsTest.java:246)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- postGetUserAuthDetailsTestUidIsError -->
  <testcase name="postGetUserAuthDetailsTestUidIsIllegalCharacters" time="0.008" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsLong" time="0.016" classname="com.userManagement.GetUserAuthDetailsTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[-1]&amp;gt; but was:&amp;lt;[0]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[-1]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.userManagement.GetUserAuthDetailsTest.postGetUserAuthDetailsTestUidIsLong(GetUserAuthDetailsTest.java:261)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- postGetUserAuthDetailsTestUidIsLong -->
  <testcase name="postGetUserAuthDetailsTestUidIsNegativeNumber" time="0.006" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsNull" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsSpace" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidIsString" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUidNonSubmissionParameters" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIllegalCharacters" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsDecimal" time="0.006" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsEmpty" time="0.007" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsError" time="0.04" classname="com.userManagement.GetUserAuthDetailsTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[-3]&amp;gt; but was:&amp;lt;[0]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[-3]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.userManagement.GetUserAuthDetailsTest.postGetUserAuthDetailsTestUserIdIsError(GetUserAuthDetailsTest.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- postGetUserAuthDetailsTestUserIdIsError -->
  <testcase name="postGetUserAuthDetailsTestUserIdIsLong" time="0.044" classname="com.userManagement.GetUserAuthDetailsTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[-1]&amp;gt; but was:&amp;lt;[0]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[-1]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.userManagement.GetUserAuthDetailsTest.postGetUserAuthDetailsTestUserIdIsLong(GetUserAuthDetailsTest.java:231)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- postGetUserAuthDetailsTestUserIdIsLong -->
  <testcase name="postGetUserAuthDetailsTestUserIdIsNegativeNumber" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsNull" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsSpace" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsString" time="0.004" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdIsZero" time="0.005" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdNonSubmissionParameters" time="0.004" classname="com.userManagement.GetUserAuthDetailsTest"/>
  <testcase name="postGetUserAuthDetailsTestUserIdNotLoggedIn" time="0.037" classname="com.userManagement.GetUserAuthDetailsTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[-3]&amp;gt; but was:&amp;lt;[0]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[-3]> but was:<[0]>
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.userManagement.GetUserAuthDetailsTest.postGetUserAuthDetailsTestUserIdNotLoggedIn(GetUserAuthDetailsTest.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- postGetUserAuthDetailsTestUserIdNotLoggedIn -->
</testsuite> <!-- Default test -->
